command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@mikebrow : `I think the purpose is to say if you have a compliant CLI it must adhere... But you could also have a different type of runtime interface such as a restful web interface. We would just have to write up a spec for that if we wanted interop over such.`

@wking said: `Until we have such an API as part of runtime-spec, I don't see how
you'd certify an HTTP-only runtime as compliant.`

From my reading, the runtime-spec does not currently require a client api. The purpose of this runtime client spec is to add a client spec for runtime implementors such as rkt and runc.  I don't think the purpose of this document is to say all runtime implementors must be command line clients and/or must include a client of this form for compliance testing. If I'm wrong ok... pls. correct me.